### PR TITLE
CI docs about 1-sided tests + default 90% CI for phi/V

### LINKED ISSUE #365: default to 95% confidence interval for effect size conversion functions?
Both in `{bayestestR}` and `{parameters}`, we have changed function argument defaults to `ci = 0.95`, but a number of test-statistic-to-effect-size conversion functions still default to `ci = 0.90`:

https://easystats.github.io/effectsize/reference/F_to_eta2.html

Screenshot 2021-08-16 at 09 33 20

Both for within-package consistency (because all other functions have `ci = 0.95`) and between-package consistency, this change will be worth it.

### PR DESCRIPTION
**Pull request by mattansb**
Close #365 

~@IndrajeetPatil Can you see if I missed any 0.95 to 0.9 changes?~ No longer relevant. But you can take a look anyway - you always catch somethig.

@bwiernik Can you look at `help("effectsize-CIs", package = "effectsize")` - the new `Choosing the Appropriate CI Level` section, see if it is clear / mistake free?

---

TODO:

- [x] extra docs - change "which level" to explain on sided CI
- [x] vignette
- [x] effectsize
  - [x] effectsize.htest
  - [ ] ~effectsize.BF?~
- [x] eta_squared
  - [ ] ~eta_squared_posterior?~
- [x] rank
- [ ] ~std_params~ https://github.com/easystats/parameters/issues/584
- [x] equi tests!
- [x] ***TESTS***:
  - [x] Printing
  - [x] 90-2 vs 95-1
  - [x] htests

### CODE REVIEW

**Comment by bwiernik** [comment]
I'm still not sure I like this approach versus using 1-tailed CIs. If we want to ensure the CI and test are consistent, I think the safest way to do that is to make the construction consistent by matching the number of tails.

I think it's as confusing to have a different alpha rate than usual as default and in the table header as it is to potentially run into a situation where 1-tailed p < .05 but 2-tailed 95% CI includes 0. I suspect a fair number of users change it to .95, potentially making opening that door for confusion. 

Or, if they leave it .90, they run the risk of reviewer pushback like you got. I can imagine this being a source of "QRP questionable research practice" accusations by a reviewer if they didn't understand the nuance here. This is probably the most problematic situation open here and many users won't be prepared to respond effectively in that situation. 

I don't really see much downside to reporting 1-tailed CIs. I agree that `[.04, Inf]` or `[.04, 1.0]` would be confusing, but if we reported it as `> .04` or  `Lower: .04` or another very different format, I think both be strictly correct and clearer that it's intentional. That could lead users to choosing consulting documentation and making an informed choice about a 2-tailed interval for these cases if they so choose.

Looking at it, I kind of like `95% CI > .04`—that is strictly accurate and not ambiguous.

**Comment by bwiernik** [comment]
Something like this perhaps:

```r
(etas <- F_to_eta2(
    f = c(40.72, 33.77, 45.31),
    df = c(2, 1, 2),
    df_error = c(18, 9, 18),
    ci = .95, tails = "lower"
))
# Eta2 (partial) |      95% CI
# ----------------------------
# 0.82           |  [ > 0.66 ]
# 0.79           |  [ > 0.49 ]
# 0.83           |  [ > 0.69]
```

**Comment by mattansb** [comment]
You're starting to convince me.... 

I'm thinking of having the argument `alternative = c("two.sided", "less", "greater")` and the output `[0.6, Inf]`, to align with what what the `*test()` functions give.

Let me think this over...

**Comment by bwiernik** [comment]
`alternative` matching arguments with `*test()` would be good.

As for `[0.6, Inf]` versus `[> 0.6]`, my main question would be whether a user would think that `Inf` was a computational failure or not. `[> 0.6]` might be clearer that this is an okay result. 

I think keeping `Inf` or `-Inf` in the data frame makes sense, but maybe `format_ci()` should change `c(.06, Inf)` to something else for display?

**Comment by mattansb** [comment]
I think a good footnote informing the one sided CI will solve any confusion.

**Comment by bwiernik** [comment]
perfect.

**Comment by mattansb** [comment]
What do you think?

``` r
library(effectsize)
cohens_d(mpg ~ am, data = mtcars, ci = 0.9)
#> Cohen's d |         90% CI
#> --------------------------
#> -1.48     | [-2.14, -0.80]
#> 
#> - Estimated using pooled SD.
cohens_d(mpg ~ am, data = mtcars, ci = 0.95, alternative = "g")
#> Cohen's d |            95% CI
#> -----------------------------
#> -1.48     | [-2.14,      Inf]
#> 
#> - Estimated using pooled SD.
#> - One-sided CIs: upper bound fixed at (Inf).
cohens_d(mpg ~ am, data = mtcars, ci = 0.95, alternative = "l")
#> Cohen's d |        95% CI
#> -------------------------
#> -1.48     | [-Inf, -0.80]
#> 
#> - Estimated using pooled SD.
#> - One-sided CIs: lower bound fixed at (-Inf).
```

<sup>Created on 2021-08-16 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by bwiernik** [comment]
Looks good. Something looks odd on the width of the positive Inf CI

**Review by bwiernik — APPROVED** [review]

**Comment by mattansb** [comment]
@bwiernik Can you take a look at the new `help("effectsize-CIs", package = "effectsize")`? I'm not sure what I wrote isn't too convoluted...

**Comment by bwiernik** [comment]
I'll make some revisions.

**Comment by mattansb** [comment]
@strengejacke does `parameters` need any updating to accommodate this change to CIs?

@DominiqueMakowski @IndrajeetPatil for `modelbased` or `correlation`?

`report` would have to mention the one sided CIs somehow.

**Comment by IndrajeetPatil** [comment]
Thanks for checking with me.

I am not going to have access to my computer again until the 30th of August, so can't really do much. 
But I _know_ that this is going to lead to tons of failing tests in `statsExpressions` for sure, so please hold off on CRAN submission until you hear from me. 🙇‍♂️

File: NEWS.md [hunk 1]
```diff
@@ -2,15 +2,18 @@
 
 ## Breaking Changes
 
+- Phi, Cohen's *w*, Cramer's *V*, ANOVA effect sizes, rank Epsilon squared, Kendall's *W* - CIs default to 95% one-sided CIs (`alternative = "greater"`).
```
Comment (IndrajeetPatil) [inline NEWS.md:5]: Maybe also mention here what the users can do to reproduce results from previous version of `effectsize`.

**Comment by bwiernik** [comment]
@mattansb I revised the new section of the `help("effectsize_CIs", "effectsize")` document to be a more straightforward. I also streamlined some grammar in the other sections and made some revisions to be more accurate (e.g., when p < alpha/2 in an F distribution, the bounds being set to 0 per Steiger isn't arbitrary, it's just that the CI with that alpha level for the ncp in this case is (0, 0) ). 

Let me know what you think.

File: R/convert_stat_chisq.R [hunk 2]
```diff
@@ -27,8 +27,9 @@
 #' \cr
 #' For adjusted versions, see Bergsma, 2013.
 #'
-#' @inheritSection effectsize-CIs Confidence Intervals
-#' @inheritSection effectsize-CIs CI Contains Zero
+#' @inheritSection effectsize_CIs Confidence (Compatibility) Intervals (CIs)
+#' @inheritSection effectsize_CIs CIs and Significance Tests
+#' @inheritSection effectsize_CIs One-Sided CIs
```
Comment (mattansb) [inline R/convert_stat_chisq.R:32]: @bwiernik I didn't originally add these as this seemed to much make the individual docs way too long for my liking. We can reference this and the troubleshooting in the same line.

Comment (mattansb) [reply R/convert_stat_chisq.R:32]: As I have placed, say, here:

https://github.com/easystats/effectsize/pull/366/files?file-filters%5B%5D=.Rd&hide-deleted-files=true#diff-e3faf4c1fa910d1c629afa2d7d19afa68c4a54b02237e0f21cd792a15d177c53R89

**Comment by mattansb** [comment]
@strengejacke I'm leaving `standardized_parameters` as-is - any supported solution would actually need to be in `{parameters} first.

**Comment by mattansb** [comment]
@IndrajeetPatil 
> But I know that this is going to lead to tons of failing tests in statsExpressions for sure, so please hold off on CRAN submission until you hear from me. 🙇‍♂️

I'm not sure when `effectsize` is schedualed to be updated. AFAIK it's not in the pipes, right?
If there should be one soon, I can hold off on the merge of this PR. If not, I'll merge... now.

**Comment by IndrajeetPatil** [comment]
You can merge now, but can wait to push it to CRAN till I am back.

Even if it was not scheduled to be updated for this cycle, I would vote for doing so given the significant number of changes that have happened since the last release. It will be good to keep the whole ecosystem in sync when we release `{easystats}` meta-package on CRAN.

**Comment by strengejacke** [comment]
> @strengejacke does parameters need any updating to accommodate this change to CIs?

Let me check...

**Comment by strengejacke** [comment]
I'm not sure where this would apply... Anova?

**Comment by IndrajeetPatil** [comment]
@mattansb You no longer need GitHub version of `{datawizard}`, since it’s on CRAN.

**Comment by mattansb** [comment]
> I'm not sure where this would apply... Anova?

ANOVA and some of the htests.

**Comment by strengejacke** [comment]
type: as as such are generally tested using 2-tailed tests and 2-sided CIs.

**Comment by mattansb** [comment]
Is that a question? I can't parse it ^_^

**Comment by strengejacke** [comment]
ups, a typo. "type" should be "typo" :-D

**Comment by strengejacke** [comment]
What I wanted to say that there's a typo in the effectsize-CIs docs

**Comment by mattansb** [comment]
lol

File: R/convert_stat_chisq.R (hunk 2 above)
Comment (bwiernik) [reply R/convert_stat_chisq.R]: Fine by me.

**Comment by bwiernik** [comment]
"as as such are generally" should be "and as such are generally" in the Docs

**Pull request merged**